dll/win32/shimgvw/lang/fr-FR.rc

```diff
@@ -4,6 +4,7 @@ STRINGTABLE
 BEGIN
     IDS_APPTITLE "Visionneuse d'Images et Fax de ReactOS"
     IDS_SETASDESKBG "Définir comme Arrière-plan du Bureau"
+    IDS_NOPREVIEW "No preview available."
```

Comment:
"Aucun aperçu n'est disponible."
![image](https://user-images.githubusercontent.com/24843587/79748465-be600a00-830d-11ea-9b69-29274ba988e1.png)

Reply:
Translations can come after this is merged. Imagine if tens of comments are added here just for translations. Let's focus on the code here.

Reply:
Translation is later.